.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  globals: {
+    fetch: true,
+	document: true,
+	localStorage: true,
+	navigator: true,
+  },
+  extends: ["airbnb-base", "prettier"],
+  plugins: ["prettier"],
+  ignorePatterns: ["bundle.js"],
+  rules: {
+	"max-classes-per-file": "off",
+	"no-console": "off",
+	"no-plusplus": "off",
+	"import/no-extraneous-dependencies": "off",
+  }
```

Comment:
```suggestion
  rules: {
      "prettier/prettier": "error",
	"max-classes-per-file": "off",
	"no-console": "off",
	"no-plusplus": "off",
	"import/no-extraneous-dependencies": "off",
  }
```
Посмотри последние изменения линтера в нашем репозитории, нужно будет еще добавить `.prettierrc`